frontend_v2/src/app/components/home/home.component.html

```diff
@@ -385,8 +385,20 @@ <h3 class="fw-light center">Partner Organizations</h3>
         </div>
       </div>
     </section>
-    <!-- Arxiv Entry -->
+
```

Comment:
let's put `<!-- twitter container -->` as comment here inorder to be consistent

Reply:
Added the comment